python/README.md

```diff
@@ -15,6 +15,26 @@
  - limitations under the License.
  -->
 
-# Iceberg
-A python implementation of the Iceberg table format.
-See the project level README for more details: https://github.com/apache/incubator-iceberg
+# Iceberg Python
+
+Iceberg is a python library for programatic access to iceberg table metadata as well as data access. The intention is to provide a functional subset of the java library.
+
+## Getting Started
```

Comment:
I think this should be in the quickstart docs as well.

Reply:
added